src/lib/y2network/interface_config_builders/bonding.rb

```diff
@@ -50,10 +50,13 @@ def bond_options
         connection_config.options
       end
 
-      # Checks if any of given device is already configured and need adaptation for bridge
+      # Returns whether any configuration of the given devices needs to be
+      # adapted in order to be added as a bonding slave
+
       # @param devices [Array<String>] devices to check
-      # @return [Boolean] true if there is device that needs adaptation
-      def already_configured?(devices)
+      # return [Boolean] true if there is a device config that needs
+      #   to be adaptated; false otherwise
+      def config_need_to_be_adapted?(devices)
```

Comment:
well, in general name is fine, I just prefer something like `require_adaptation?`